modules/tinymce/src/plugins/table/main/ts/selection/CellSelection.ts

```diff
@@ -211,6 +203,7 @@ export default function (editor: Editor, lazyResize: () => Option<TableResize>,
 
   const destroy = function () {
     handlers.each(function (_handlers) {
+      // TODO: do we need these? can this be deleted? destroy() is only used in one place atm
```

Comment:
see comment

Reply:
Yeah, I don't think this is needed either. All TinyMCE events are simulated so there's no need to unbind them, so it's only if we bind directly to an element.

Reply:
- [ ] delete